src/foam/nanos/u2/navigation/UserInfoNavigationView.js

```diff
@@ -71,7 +71,7 @@ foam.CLASS({
             .add(this.slot((agent, user) => {
               return this.E().addClass('name-container')
                   .start('span').addClass(this.myClass('userName'))
-                    .add( agent ? user.businessName() : user.label())
+                    .add( agent.firstName ? user.businessName : user.label())
```

Comment:
Why aren't we just using `label()`?
```suggestion
                    .add(user.label())
```

If it's the portal, then `agent` isn't there and `user` is a `User`, so `user.label()` will give us the legal name (human name), which is what we want.

If it's Ablii, then `agent` _is_ there and `user` is a `Business`, so `user.label()` will give us the business name, which is what we want.